app/scripts/views/form.js

```diff
@@ -166,7 +166,14 @@ define(function (require, exports, module) {
       // applied, no submit handler is fired, and the form validation does not
       // take place.
       if (this.isFormEnabled()) {
-        this.$('button[type=submit]').addClass('disabled');
+        if (this.isInExperimentGroup && this.isInExperimentGroup('disabledButtonState', 'treatment')) {
```

Comment:
Is this logic supposed to be gated to a particular set of views? This looks like it's going to affect *all* views that have the experiment-mixin.

That reminds me, for this to take effect on the signin page, you need to add the ExperimentMixin.